content/en/docs/Kyverno CLI/_index.md

```diff
@@ -224,3 +224,159 @@ policies:
 ```
 kyverno apply /path/to/add_network_policy.yaml --resource /path/to/required_default_network_policy.yaml -f /path/to/value.yaml
 ```
+
+#### Policy Report
+
+Use '--policy_report' with apply command to generate policy report.
```

Comment:
"Use `--policy_report` with the `apply` command to generate a policy report."

Reply:
Can you also write a sentence or two for what purpose this section serves? Why would a user want to get a policy report from the CLI? What good is it?